Xcode-Extension/DefaultsCommand.swift

```diff
@@ -114,22 +121,27 @@ struct DefaultKey {
     /// key name in api response
     public let key: String?
 
-    public init(_ name: String, type: DefaultType, options: Option = .none, default value: Any? = nil, key: String? = nil) {
+    public let scope: Scope
+
+    public init(_ name: String, type: DefaultType, options: Option = .none, default value: Any? = nil, key: String? = nil, scope: Scope) {
         self.name = name
         self.type = type
         self.options = options
         self.default = value
         self.key = key
+        self.scope = scope
     }
 }
 
 
 extension DefaultKey {
     var prefKey: String {
+        let prefix = scope != .common ? (scope.rawValue + ".") : ""
```

Comment:
Isn't `prefix` a Swift literal?

Reply:
@GermanVelibekovHouzz it compiled and tested so I guess it's ok